From 60a1917825044fc3a43dc39ef8de2b944150dc8a Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Stefan=20Br=C3=BCns?= <stefan.bruens@rwth-aachen.de>
Date: Sat, 25 Jun 2022 18:50:45 +0200
Subject: [PATCH 1/4] Only import attrdict where needed

Its only used for MSVC builds, but an unnecessary external dependency
elsewhere.
---
 buildtools/config.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/buildtools/config.py b/buildtools/config.py
index 4482ba9d..493a63c5 100644
--- a/buildtools/config.py
+++ b/buildtools/config.py
@@ -27,8 +27,6 @@ from distutils.dep_util  import newer
 
 import distutils.sysconfig
 
-from attrdict import AttrDict
-
 runSilently = False
 
 #----------------------------------------------------------------------
@@ -989,6 +987,8 @@ def getMSVCInfo(PYTHON, arch, set_env=False):
     if MSVCinfo is not None:
         return MSVCinfo
 
+    from attrdict import AttrDict
+
     # Note that it starts with a monkey-patch in setuptools.msvc to
     # workaround this issue: pypa/setuptools#1902
     cmd = \
-- 
2.36.1

